WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -5,16 +5,31 @@ import WordPressFlux
 
 class NoticePresenter: NSObject {
     private let store: NoticeStore
-    private let presentingViewController: UIViewController
+    private let window: UntouchableWindow
+    private var view: UIView {
+        guard let view = window.rootViewController?.view else {
+            fatalError("Root view controller shouldn't be nil")
+        }
+        return view
+    }
     private var currentContainer: NoticeContainerView?
 
     let generator = UINotificationFeedbackGenerator()
 
     private var storeReceipt: Receipt?
 
-    private init(store: NoticeStore, presentingViewController: UIViewController) {
+    private init(store: NoticeStore) {
         self.store = store
-        self.presentingViewController = presentingViewController
+
+        let windowFrame: CGRect
+        if let mainWindow = UIApplication.shared.keyWindow {
+            windowFrame = mainWindow.offsetToAvoidStatusBar()
+        } else {
+            windowFrame = .zero
+        }
+        window = UntouchableWindow(frame: windowFrame)
+        window.windowLevel = .alert
```

Comment:
This is causing problems with some system UI. The examples I have found are the share sheet and the 1Password extension. Here are steps to reproduce:

1. Go to any post in Reader
2. Open the share sheet and try to share by any means (I tried iMessage)
3. The iMessage/Email/etc. button isn't pressable

I was able to fix this by changing this line to `window.windowLevel = .alert - 0.1`. Its obviously very hacky so may be worth thinking of alternatives. The other "fix" would be to only display the window when its needed (should remove the bug but could be acceptable).

`UIWindow`s are always full of surprises 😅